### PR TITLE
--entrypoint= should delete existing entrypoint

### PR DESCRIPTION
**Pull request by baude**
Resolves: #572

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
LGTM

File: cmd/podman/create.go [hunk 1]
```diff
@@ -547,7 +547,10 @@ func parseCreateOpts(c *cli.Context, runtime *libpod.Runtime, imageName string,
 	if len(entrypoint) == 0 {
 		entrypoint = data.ContainerConfig.Entrypoint
 	}
-
+	// if entrypoint=, we need to clear the entrypoint
```
Comment (TomSweeneyRedHat) [inline cmd/podman/create.go]: nit comment has 'entrypoint=,'  Should it be "If the entrypoint has no value" or perhaps entrypoint is nil?

Comment (baude) [reply cmd/podman/create.go]: only if you want me to try and pass all the tests again :)

**Comment by mheon** [comment]
@rh-atomic-bot r+

File: cmd/podman/create.go (hunk 1 above)
Comment (TomSweeneyRedHat) [reply cmd/podman/create.go]: Touch up in some unrelated future PR is fine by me, I trust ya.

**Comment by redbaron** [comment]
Hmm , maybe remove "Closes:" from merge commit created by robot? Then PRs would show as "Merged" in github interface

**Pull request closed without merging**